test/helpers/helpers.test.js

```diff
@@ -265,4 +265,29 @@ describe('the deploy function', () => {
     expect(mockDeployProject.mock.calls[0][0].serviceSid).toBe(undefined);
     expect(mockDeployProject.mock.calls[0][0].serviceName).toBe(APP_NAME);
   });
+
+  it('should display an error when the API key is not provided', () => {
+    return expect(
+      deploy.call({
+        twilioClient: {
+          username: 'testAccountSid',
+          password: 'testAuthToken',
+          accountSid: 'testAccountSid',
+        },
+        flags: {},
+      })
+    ).rejects.toMatchInlineSnapshot(`
+[Error: No API Key found.
+
+Please login to the Twilio CLI to create an API key:
+
+twilio login
+
+Alternatively, the Twilio CLI can use credentials stored in these environment variables:
+
+TWILIO_ACCOUNT_SID = your Account SID from twil.io/console
+TWILIO_API_KEY = an API Key created at twil.io/get-api-key
+TWILIO_API_SECRET = the secret for the API Key]
```

Comment:
Nit: Can we refactor the expected error message to a constant so the test and implementation reference the same thing?

Reply:
I tried that and it didn't work. Notice that brackets surround the output, but not the input. The brackets don't show up in the console, but there is some special formatting.

Even if it did work, I don't think it's necessary because I used `toMatchInlineSnapshot `. The test highlighted in the snippet above was generated by jest. If the error message ever changes, we update the test by pressing the `u` key when running the tests in watch mode. We won't have to update this test manually.

https://jestjs.io/docs/en/snapshot-testing#inline-snapshots